lib/map/security_token/configuration.rb

```diff
@@ -78,6 +90,21 @@ def client_assertion_certificate
         OpenSSL::X509::Certificate.new(File.read(client_cert_path))
       end
 
+      def public_jwks
```

Comment:
I think I prefer logic in `service.rb`. I think the configuration should be as much a collection of static objects as possible